includes/class-sensei-lesson.php

```diff
@@ -443,6 +446,24 @@ public function handle_get_prerequisite_meta_box_content() {
 		wp_die(); // This is required to terminate immediately and return a proper response.
 	}
 
+	/**
+	 * Display the Video settings panel for the lesson.
+	 *
+	 * @return void
+	 */
+	public function lesson_video_meta_box_content() {
+		$html  = '<p>' . esc_html__( 'Control how students progress through the course based on their interactions with lesson video.', 'sensei-lms' ) . '<p>';
+
+		$lesson_course = get_post_meta( get_the_ID(), '_lesson_course', true );
+		if ( $lesson_course ) {
+			$edit_course_url = get_edit_post_link( $lesson_course );
```

Comment:
`get_edit_post_link` builds a link to the editor of the course. 
However, I failed finding a way how to make Course sidebar open and Video section expanded.
Is there any straight way of doing it?